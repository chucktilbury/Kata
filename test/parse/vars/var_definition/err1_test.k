;-------------------------------------------------------------------------------
; verify that invalid assignment item produces error
;-------------------------------------------------------------------------------

var string some_name = dict
