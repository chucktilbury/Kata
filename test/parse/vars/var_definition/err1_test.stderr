Syntax: /home/chuck/Src/Kata/test/parse/vars/var_definition/err1_test.k: 6: 1: expected an assignment item but got dict
Syntax: /home/chuck/Src/Kata/test/parse/vars/var_definition/err1_test.k: 6: 1: expected module item or end of file but got var
Syntax: /home/chuck/Src/Kata/test/parse/vars/var_definition/err1_test.k: 6: 1: module cannot be empty
